test/net/sourceforge/kolmafia/extensions/ClearSharedState.java

```diff
@@ -12,15 +12,15 @@
 import net.sourceforge.kolmafia.preferences.Preferences;
 import net.sourceforge.kolmafia.request.GenericRequest;
 import net.sourceforge.kolmafia.textui.command.AbstractCommand;
-import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
 import org.junit.jupiter.api.extension.ExtensionContext;
 
-public class ClearSharedState implements AfterAllCallback {
+public class ClearSharedState implements BeforeAllCallback {
 
   // Prevents leakage of shared state across test classes.
 
   @Override
-  public void afterAll(ExtensionContext context) {
+  public void beforeAll(ExtensionContext context) {
```

Comment:
I went for afterAll because I could imagine a case where a test prepopulated a directory with files before tests and beforeAll could break that.  If there is a test specific before/after that can still be done and, realistically, it's not highly probable so...

Reply:
BeforeAll is before a whole test class though right? We shouldn't be encouraging any statefulness between test classes